src/internal/pdf/pdf_test.go

```diff
@@ -0,0 +1,21 @@
+package pdf_test
+
+import (
+	"main/internal/pdf"
+	"path/filepath"
+	"testing"
+)
+
+// Test a simple label with input that should definitely work
+func TestPdfGeneration_Simple(t *testing.T) {
+	labelText := "Hocked bean soup"
+	filePathName, err := filepath.Abs("./tmp/testOutput.pdf")
+	if err != nil {
+		t.Fatal("Unable to get Abs filepath")
```

Comment:
I'm expecting using `t.Fatal` will completely stop test execution. For now, it's okay, but it might be problematic if you have other test suites in this same file. Then if your first test fails, the second tests are not even run, which is not nice when you are running tests in CI and want to see which tests passed and which failed.

https://github.com/smartystreets/goconvey might help here, and it also gives nicer error messages.

When you use goconvey, you can write:
```
			So(err, ShouldBeNil)
```

Reply:
I rewrote each of my `t.Fatal` calls with a combination of `t.Log` and `t.Fail`.

I also made issue #4 to address your suggestion to start using Goconvey!